GTFO-API/Patches/GameDataInit_Patches.cs

```diff
@@ -10,9 +11,29 @@ internal class GameDataInit_Patches
     {
         [HarmonyPatch(nameof(GameDataInit.Initialize))]
         [HarmonyWrapSafe]
-        [HarmonyPrefix]
-        public static void Initialize_Prefix()
+        [HarmonyPostfix]
+        public static void Initialize_Postfix()
         {
+            Analytics.enabled = false;
+
+            GameSetupDataBlock setupBlock = GameDataBlockBase<GameSetupDataBlock>.GetBlock(1);
+            if (setupBlock?.RundownIdToLoad != 1)
+            {
+                APILogger.Verbose(nameof(GameDataInit_Patches), $"RundownIdToLoad was {setupBlock.RundownIdToLoad}. Setting to 1");
+                RundownDataBlock.RemoveBlockByID(1);
+
+                RundownDataBlock block = RundownDataBlock.GetBlock(setupBlock.RundownIdToLoad);
+                block.persistentID = 1;
+                block.name = $"MOVEDBYAPI_{block.name}";
+
+                RundownDataBlock.RemoveBlockByID(setupBlock.RundownIdToLoad);
+                RundownDataBlock.AddBlock(block, -1);
```

Comment:
Extremely rare case, but if you have both id 1 RundownDataBlock and non-RundownIdToLoad id 1 GameSetupDataBlock it will return duplicate key exception.

Reply:
I remove the block with id 1 at L23 to make sure no overlaps will occur

Reply:
RemoveBlockByID method doesn't update their id-lookup dictionaries
```cs
public static uint AddBlock(T block, int addIndex = -1)
{
    ...
    GameDataBlockBase<T>.s_blockIDByName.Add(block.name, block.persistentID);
    GameDataBlockBase<T>.s_dirtyBlocks.Add(block.persistentID);
    GameDataBlockBase<T>.s_blockByID.Add(block.persistentID, block);
    return block.persistentID;
}
```cs
public static void RemoveBlockByID(uint id)
{
    T block = GameDataBlockBase<T>.GetBlock(id);
    GameDataBlockBase<T>.Wrapper.Blocks.Remove(block);
}
```